doc/api.md

```diff
@@ -104,7 +105,7 @@ Authorization specifies a subset of the Authorization struct, that is safe for u
 | Field | Description | Scheme | Required |
 | ----- | ----------- | ------ | -------- |
 | type | Set the authentication type. Defaults to Bearer, Basic will cause an error | string | false |
-| credentials | The secret's key that contains the credentials of the request | *[v1.SecretKeySelector](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.23/#secretkeyselector-v1-core) | false |
+| credentials | The secret's key that contains the credentials of the request | *corev1.SecretKeySelector | false |
```

Comment:
Interesting that the link got dropped. Core is just an alias...I wonder why that got carried forward here.

Reply:
Huh, probably a bug in the underlying doc generator, which seems like it has been deleted and no longer maintained.